### PR TITLE
Add contributor issue template, add info abuot project tech team to readme

### PR DESCRIPTION
**Pull request by lorenanicole**
# Scope

This PR adds:
- contributor issue template for project tech infra
- add info about project tech team to readme
- add project tech infra team list in `PROJECT_TECH_TEAM.md`

### CODE REVIEW

File: PROJECT_TECH_INFRA.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Name | GitHub Handle | PyLadies Chapter | Team Role |
+| --| --| --| --|
+| Mariatta | @Mariatta  | PyLadies Vancouver | Co-Lead |
+| Bethany G | @BethanyG   | PyLadies Oakland | Co-Lead |
+| Tania Allard | @trallard  | PyLadies Northwestern UK | Co-Lead |
+| Lorena Mesa* | @lorenanicole | PyLadies Chicago | Member |
+| Carol Willing | @willingc   | PyLadies Core | Member |
+| Jackie Kazil | @jackiekazil   | PyLadies DC | Member |
+| Flávia Cardoso | @fcardoso0917  | PyLadies Uruguay | Member |
+Amadikwa Joy N | @treasurechristain | Pyladies IMO | Member |
```
Comment (Mariatta) [inline PROJECT_TECH_INFRA.md]: missing | at start
```suggestion
| Amadikwa Joy N | @treasurechristain | Pyladies IMO | Member |
```

Comment (Mariatta) [reply PROJECT_TECH_INFRA.md]: Can we also ask for their Slack handle? I will add all team members into a Slack group.

File: PROJECT_TECH_INFRA.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* @lorenanicole is a PyLadies admin that can help support the co-leads as needed.
+
+# Contributing
+
+If you would like to join, open an issue requesting to join using the [contributor template](https://github.com/pyladies/pyladies/issues/new).
```
Comment (Mariatta) [inline PROJECT_TECH_INFRA.md]:
```suggestion
If you would like to join, open an issue requesting to join using the [contributor template](https://github.com/pyladies/pyladies/issues/new). Please also join our discussions in our Slack, `#project-tech-infra` channel.
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
-# Overview
+# PyLadies 
+
+This website is managed by the [PyLadies Tech and Infra team](PROJECT_TECH_INFRA.md). If you would like to join as an official member, read more here! This website welcomes pull requests from anyone in the community.
```
Comment (Mariatta) [inline README.md]:
```suggestion
This website is managed by the [PyLadies Tech and Infra team](PROJECT_TECH_INFRA.md). If you would like to join as an official member, read more here! You can also join us in Slack, `#project-tech-infra` channel.
This website welcomes pull requests from anyone in the community. 
```

File: .github/ISSUE_TEMPLATE/contributors.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+labels: contributor
+assignees: pyladies/pyladies-global-admin
+
+---
+
+_This template should be used (edited) for PyLadies members that wish to join the Project Tech & Infra team.
+
+**Proposed Team Member Details**
+
+Name | GitHub Handle | PyLadies Chapter | Team Role
```
Comment (Mariatta) [inline .github/ISSUE_TEMPLATE/contributors.md]:
```suggestion
Name | GitHub Handle | Slack Handle | PyLadies Chapter | Team Role 
```

File: .github/ISSUE_TEMPLATE/contributors.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+assignees: pyladies/pyladies-global-admin
+
+---
+
+_This template should be used (edited) for PyLadies members that wish to join the Project Tech & Infra team.
+
+**Proposed Team Member Details**
+
+Name | GitHub Handle | PyLadies Chapter | Team Role 
+| --| --| --|
```
Comment (Mariatta) [inline .github/ISSUE_TEMPLATE/contributors.md]:
```suggestion
| --| --| --| --|
```

File: PROJECT_TECH_INFRA.md (hunk 1 above)
Comment (lorenanicole) [reply PROJECT_TECH_INFRA.md]: Sure why not

**Pull request merged**